deploy/crds/shipwright.io_buildruns.yaml

```diff
@@ -104,6 +104,9 @@ spec:
                     description: 'Name of the referent; More info: http://kubernetes.io/docs/user-guide/identifiers#names'
                     type: string
                 type: object
+              stateOverride:
```

Comment:
Needs regenerated after the below update of the json.

Reply:
swore I did this .... hmmm